### PR TITLE
fix-up  code  bug

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

**Comment by tanjunchen** [comment]
@hwdef

File: tests/e2e/manifests/verify_manifest_lists.go [hunk 1]
```diff
@@ -437,10 +437,8 @@ func verifyManifestList(manifest, imageName, tag string) error {
 		return fmt.Errorf("not a manifest list: %s", ml.MediaType)
 	}
 	aList := make([]string, len(archList))
-	for i := range archList { // copy into a temp slice.
-		aList[i] = archList[i]
-	}
-
+	// copy into a temp slice.
+	copy(aList,archList)
```
Comment (neolit123) [inline tests/e2e/manifests/verify_manifest_lists.go]: please add space after `aList,`.

Comment (tanjunchen) [reply tests/e2e/manifests/verify_manifest_lists.go]: OK  Thank You

**Pull request merged**